src/status_im/chat/handlers.cljs

```diff
@@ -28,16 +28,18 @@
     (messages/delete-by-chat-id id)))
 
 (defn delete-messages!
-  [{:keys [current-chat-id]} [_ chat-id]]
-  (let [id (or chat-id current-chat-id)]
-    (messages/delete-by-chat-id id)))
+  [{:keys [current-chat-id chats]} [_ chat-id]]
+  (let [id                   (or chat-id current-chat-id)
+        {:keys [group-chat]} (chats/get-by-id chat-id)]
+    (when group-chat
```

Comment:
I don't see why we should treat deletion differently depending on if it is group or 1on1 chat. To me they should do the same thing, which is to mark messages as deleted (which event/view should respect).

Reply:
@oskarth In current implementation group messages are getting removed on chat deletion. We cannot do same thing with 1-1 chat messages because old offline messages would be treated as new and would show deleted chat back to the user.
Storing deleted messages UUIDs for that purpose might be the solution, but it requires additional work. This PR was expected as a quick bugfix for the upcoming release.

Reply:
> This PR was expected as a quick bugfix for the upcoming release.

Gotcha, let's make it prio to merge it then